xml/System/Activator.xml

```diff
@@ -197,7 +197,11 @@
           <paramref name="typeName" /> or <paramref name="assemblyName" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">
           <paramref name="assemblyName" /> is the empty string ("").</exception>
-        <exception cref="T:System.IO.PathTooLongException">An assembly or module was loaded twice with two different evidences, or the assembly name is longer than MAX_PATH characters.</exception>
+        <exception cref="T:System.IO.PathTooLongException">An assembly or module was loaded twice with two different evidences.
```

Comment:
Should this include the warning for "exceeds the system-defined max path length"?

Reply:
It does four lines below, @BillWagner. Since they're independent conditions, I separated them with an "-or-".